public/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
Good call here! Just a note - if others are using a different box sizing, this will override their CSS since it's using the universal selector.
I think it's unlikely that anyone will be using anything other than `border-box`, but it's a good thing to be aware of.